pipeline/clean/merge-mono.sh

```diff
@@ -13,13 +13,24 @@ output=$1
 max_sent=$2
 datasets=( "${@:3}" )
 
-dir=$(dirname "${output}")
-mkdir -p "${dir}"
-
-pigz -dc "${datasets[@]}" |
-  ${BIN}/dedupe |
-  shuf -n "${max_sent}" |
-  pigz >"${output}"
+LOCALE="${LOCALE:-}"
```

Comment:
The changes to this file essentially mean we have two different scripts in one (because all of the actual work commands are duplicated). This isn't great, because it means they're likely to get out of sync at some point.

Is there a reason you can't pass args similar to the existing pipeline script, and avoid the if/else? It looks to me like this ends up getting passed things like:
```
 + output=/data/rw/andre/data/ca-en/dev/clean/mono.ca.gz
 + max_sent=100000000
 + datasets=("${@:3}")
```

As far as I can tell, the zstd files present on in `$MOZ_FETCHES_DIR` in your tasks should all contain `$LOCALE`, so I'm not sure why we need to filter that. And you should be able to set the full output file name in the kind as well?

Let me know if I'm missing something here.